### PR TITLE
Created login page

### PR DESCRIPTION
**Pull request by dhairyabaweja**
Solves issue #1

### CODE REVIEW

**Comment by Jigar3** [comment]
@dhairyabaweja You have created two PR's with the same commits. The mistake you did was editing the code on the same branch. It is advisable to solve seperate issues on seperate branches, this is beacuse commits that solve one issue appear on PR for the other issue.

**Comment by Jigar3** [comment]
You could create a new branch, then use
`git reset HEAD~<no of commits to go back>`
This command will reset the branch to a previous commit. You can now go back to your commit and edit it and send a new PR from that branch

File: artworks.md [hunk 1]
```diff
@@ -7,7 +7,6 @@
 5. Archies11 : https://codepen.io/archies11/pen/LMMWYg
 6. KeshavSharma21099:- https://codepen.io/keshavsharma21099/pen/MZZKWv
 7. Rajatsharma1007: https://codepen.io/rajatsharma1007/pen/MZZmPM
-8. BLJS2warchief (Balaji Meruva) : https://codepen.io/BLJS2warchief/full/qLeQJw
```
Comment (Jigar3) [inline artworks.md]: Don't remove this

Comment (dhairyabaweja) [reply artworks.md]: I m closing this PR and making a new one , I m having a lot of confusion..

**Pull request closed without merging**